createUserFeatureFlag/main.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/models"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/go-playground/validator/v10"
+)
+
+var validate *validator.Validate
+
+func init() {
+	validate = validator.New()
+}
+
+func processPutById(userId string, flagId string, featureFlagUserMapping models.FeatureFlagUserMapping) (*models.FeatureFlagUserMapping, error) {
+
+	db := database.CreateDynamoDB()
+
+	item, err := dynamodbattribute.MarshalMap(featureFlagUserMapping)
+	if err != nil {
+		return nil, err
+	}
+
+	input := &dynamodb.PutItemInput{
+		TableName:           aws.String(database.GetTableName(utils.FFUM_TABLE_NAME)), //TODO
+		Item:                item,
+		ConditionExpression: aws.String("attribute_not_exists(userId) OR attribute_not_exists(flagId)"),
+	}
+
+	_, err = db.PutItem(input)
+	if err != nil {
+		utils.DdbError(err)
+		return nil, err
+	}
+
+	return &featureFlagUserMapping, nil
+}
+
+func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	userId, found := req.PathParameters["userId"]
+	if !found {
+		log.Panic("User ID not passed")
+		return utils.ClientError(http.StatusBadRequest, "User ID not passed in request")
+	}
+
+	flagId, found := req.PathParameters["flagId"]
+	if !found {
+		log.Panic("Flag ID not passed")
```

Comment:
where are we handling the Panic?

Reply:
Remove the found check as discussed here https://github.com/Real-Dev-Squad/feature-flag-backend/pull/32#discussion_r1194438261